### PR TITLE
Add corner cases integration test

### PR DESCRIPTION
**Pull request by bartcharbon**
### Before merge:
- [ ] Functionality works & meets specs
- [ ] No issues running `bash test/pipeline_test.sh`
- [ ] Code reviewed
- [ ] Documentation was updated

### After merge:
- [ ] Added feature/fix to draft release notes

## Notes by PR author
None.

### CODE REVIEW

File: test/test.sh [hunk 1]
```diff
@@ -270,6 +284,11 @@ run_tests () {
   test_snv
   after_each
 
+  TEST_ID="test_corner_cases"
+  before_each
+  test_snv
```
Comment (dennishendriksen) [inline test/test.sh:289]: test_snv --> test_corner_cases

**Review by dennishendriksen — CHANGES_REQUESTED** [review]

File: test/test.sh (hunk 1 above)
Comment (dennishendriksen) [reply test/test.sh]: fixed by [4f3a03c](https://github.com/molgenis/vip/pull/269/commits/4f3a03c9da5dabe6a573d6763f30facc419694ef)

**Review by dennishendriksen — APPROVED** [review]
